demisto_sdk/__main__.py

```diff
@@ -589,10 +591,27 @@ def id_set_command(**kwargs):
     id_set_creator.create_id_set()
 
 
+# ====================== find-dependencies ====================== #
+@main.command(name="find-dependencies",
```

Comment:
add `hidden=True

Reply:
But it is not internal command.

Reply:
I do not think contributors should run this command,
but that we should add it automatically to the their pack_metada.

Reply:
Hmm it is mainly for contributors, they should define what dependencies are mandatory or not after running this command.
@yuvalbenshalom how do you think we should handle it?

Reply:
@IkaDemisto I think @bakatzir is correct. we need a command that dose "update dependencies", and update the metadata.
we can consider to split it. lets talk about it.

Reply:
we cannot add this automatically, when you init pack there are no dependencies

Reply:
will create the updating command in a different issue and PR